open_feature/exception/exceptions.py

```diff
@@ -85,3 +85,37 @@ def __init__(self, error_message: str = None):
         @return: the generic TypeMismatchError exception
         """
         super().__init__(error_message, ErrorCode.TYPE_MISMATCH)
+
+
+class TargetingKeyMissingError(OpenFeatureError):
+    """
+    This exception should be raised when the provider requires a targeting key
+    but one was not provided in the evaluation context.
+    """
+
+    def __init__(self, error_message: str = None):
+        """
+        Constructor for the TargetingKeyMissingError. The error code for this type of
+        exception is ErrorCode.TARGETING_KEY_MISSING.
+        @param error_message: a string message representing why the error has been
+        raised
+        @return: the generic TargetingKeyMissingError exception
+        """
+        super().__init__(error_message, ErrorCode.TARGETING_KEY_MISSING)
+
+
+class InvalidContextError(OpenFeatureError):
+    """
+    This exception should be raised when The evaluation context does not meet provider
```

Comment:
Being very picky now but there's a minor typo here

```suggestion
    This exception should be raised when the evaluation context does not meet provider
```

Reply:
Fixed